src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -59,6 +59,19 @@
 
     <ArtifactsLogRepoDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsLogDir)', '$(RepositoryName)'))</ArtifactsLogRepoDir>
     <PackageReportRepoDir>$([MSBuild]::NormalizeDirectory('$(PackageReportDir)', '$(RepositoryName)'))</PackageReportRepoDir>
+
+    <RidAgnosticVerticalName>Windows_x64</RidAgnosticVerticalName>
+
+    <!--
+      Publish only RID-specific assets when we're not doing source-build and one of the following cases is true:
+      - This is not the Rid-agnostic vertical
+      - This is a BuildPass1 build
+    -->
+    <EnableDefaultRidSpecificArtifacts>false</EnableDefaultRidSpecificArtifacts>
```

Comment:
Do we need to flow this property when it's false? Asking as reducing the number of properties that we flow to the inner build is preferred.

Reply:
OK. Consider moving this to runtime.proj and add a tracking issue to remove it.